tests/Tester.cs

```diff
@@ -26,13 +28,13 @@ private static ChargeClient CreateChargeClient()
             return new ChargeClient(uri, Network, allowInsecure: true);
         }
 
-        private static LndClient CreateLndClient()
+        internal static LndClient CreateLndClient()
         {
             var host = CommonTests.Docker ? "http://lnd:8080" : "http://127.0.0.1:32736";
             return new LndClient(new LndRestSettings { AllowInsecure = true, Uri = new Uri(host) }, Network.RegTest);
         }
 
-        private static LndClient CreateLndClientDest()
+        internal static LndClient CreateLndClientDest()
```

Comment:
Needed to make them available to the test setup methods to connect the LNDhub instances. See the comment for `EnsureConnectedToDestinations`.